laserstudio/__main__.py

```diff
@@ -68,4 +69,8 @@
     win = LaserStudio(yaml_config)
     win.setWindowTitle(app.applicationDisplayName())
     win.show()
-    sys.exit(app.exec())
+    return app.exec()
```

Comment:
This does not forward error codes from Qt to the shell, like `sys.exit` does.
Maybe you want the caller of `main` to call `sys.exit` with the result returned from `main`?

Reply:
When building with poetry, the entrypoints are generated with wrapping 'sys.exit', so main() should be called with sys.exit() wrapping, and it is indeed not the case in line 76.